sdk/include/crt/intrin.h

```diff
@@ -11,6 +11,7 @@
 #if defined(_M_IX86) || defined(_M_X64)
 //#include <immintrin.h>
 //#include <ammintrin.h>
+#include <xmmintrin.h>
```

Comment:
This is not included from native headers afaics

Reply:
Yeah for me it was included somewhere deep inside immintrin.h,
but since that was disabled here (and I don't think we have it)....